src/config/env/dev.json

```diff
@@ -22,6 +22,7 @@
   "ENS_GATEWAY": "limo",
   "RESOLVER_CONTRACT_ADDRESS": "0x4B1488B7a6B320d2D721406204aBc3eeAa9AD329",
   "CONTROLLER_CONTRACT_ADDRESS": "0x6ff05b6271bbed8f16a46e6073d27ad94224e0ac",
+  "CONTROLLER_V2_CONTRACT_ADDRESS": "0x055a4d2c8f044bc9aac260c9194fd177654329cd",
```

Comment:
is it ok if this address is the same as the mainnet?

Reply:
It has not been deployed yet to mainnet, will be updated once it does